src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,93 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    static final int VALID_AGE = 45;
+    static final int INVALID_AGE1 = 15;
+    static final int INVALID_AGE2 = 180;
+    static final String VALID_PASSWORD = "qwerty";
+    static final String INVALID_PASSWORD = "1234";
```

Comment:
I'm not confident if we should use constants in our tests

Reply:
I saw constants in tests to another tasks